browser/lib/markdown-it-fence.js

```diff
@@ -12,7 +12,7 @@ module.exports = function (md, renderers, defaultRenderer) {
     }
 
     const marker = state.src.charCodeAt(pos)
-    if (!(marker === 96 || marker === 126)) {
+    if (marker !== 0x7E/* ~ */ && marker !== 0x60 /* ` */) {
```

Comment:
Do you mind to add a coment explaining what the chars are you're using as markers?

Reply:
don't mind my comment.. i wanted you to write that `0x7E` means `~` and `0x60` `\`` but you already did that and i simply didn't understand your code.. I'm sorry for the supid comment :)

Reply:
Maybe something like:
```
// If the first character is not "open code blocks character " then return false
```
Seem redundant hahaha :smile:

Reply:
hm.. i think you can just ignore my initial question and everything is fine :)